Form/Type/ActionsType.php

```diff
@@ -12,6 +12,7 @@
 namespace Integrated\Bundle\ChannelBundle\Form\Type;
 
 use Integrated\Common\Form\Type\ActionsType as BaseActionsType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
```

Comment:
Probley has something todo with the fact that the class that is extended from is not compatible with 2.8.